Modules/Role/Http/Services/RoleService.php

```diff
@@ -43,32 +39,24 @@ public function submitRole(array $requestData): array
      */
     public function show(): array
     {
-        $role = Role::where('role', '!=', 'admin')
-            ->where('role', '!=', 'Content Moderator')
+        $role = Role::where('id', '!=', '1')
```

Comment:
Instead of using id, Declare admin as constant in role model as we did before for other roles. And use that constant in condition.

Reply:
Fixed